apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -16,6 +16,8 @@ class Full_Site_Editing {
 	 */
 	private static $instance = null;
 
+	private $template_post_types = [ 'wp_template', 'wp_template_part', 'wp_template_part_type' ];
```

Comment:
`wp_template_part_type` is probably not needed here? Technically it's not a post type but a taxonomy. Currently we have `wp_template` and `wp_template_part` post types, and `wp_template_type` and `wp_template_part_type` taxonomies that correspond to them.

Reply:
Fixed on 3fb1b9e875497faed8151f6a4c4a9c9ff9d815e0.